cdist/conf/type/__link/explorer/state

```diff
@@ -52,7 +53,9 @@ case "$type" in
          echo sourcemissing
          exit 0
       fi
+      # TODO SC2012: use stat?
       destination_inode=$(ls -i "$destination" | awk '{print $1}')
```

Comment:
`stat --format '%i' "$destination` is perfect and saves one fork (`stat -f '$i'` on FreeBSD)

Reply:
Thinking about this it's probably not worth the effort to change this code as `/usr/bin/stat` does not seem to be POSIX. Adapting for all supported OSes make the code harder to read and maintain so we are probably better with disabling SC2012 in this case (instead of using `stat`).
This does _not_ affect the `readlink`-replacement.

Reply:
@tom-ee I was thinking the same.